catalog/video/video-admin-plugin/src/test/java/org/codice/alliance/video/ui/StreamMonitorHelperTest.java

```diff
@@ -157,10 +150,11 @@ public void testCallStartAndStopMonitoringStream() {
   @Test
   public void testNetworkInterfaces() throws SocketException {
 
-    Inet4Address inetAddress = PowerMockito.mock(Inet4Address.class);
+    Inet4Address inetAddress = mock(Inet4Address.class);
 
-    NetworkInterface networkInterface = PowerMockito.mock(NetworkInterface.class);
+    NetworkInterface networkInterface = mock(NetworkInterface.class);
 
+    when(inetAddress.getHostAddress()).thenReturn("0.0.0.0");
```

Comment:
just a heads up but Sonar might complain about this since its the same string literal used throughout the same file and would recommend refactoring using a Constant

Reply:
Oh, good catch. I'll look at that.